functions-and-scope.js

```diff
@@ -53,6 +90,29 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // averageGrade([6, 4, 5]) geeft 5
 // averageGrade([8, 9, 4, 6, 10]) geeft 7.4
 
+// Stap 1: Schrijf de functie averageGrade
+function averageGrade(cijfers) {
+    // Stap 2: Maak een variabele om de som bij te houden
+    let som = 0;
+
+    // Stap 3: Loop door de array en bereken de totale som
+    for (let i = 0; i < cijfers.length; i++) {
+        som += cijfers[i];
+    }
+
+    // Stap 4: Bereken het gemiddelde door de som te delen door het aantal cijfers
+    let gemiddelde = som / cijfers.length;
```

Comment:
Je kan hier direct het gemiddelde retourneren in plaats van een variabele aan te maken die je vervolgens retourneerd.